tests/e2e/specs/keplr/keplr-spec.js

```diff
@@ -33,6 +33,14 @@ describe('Keplr', () => {
       });
     });
 
+    it(`should disconnect the wallet from all the connected DAPPs`, () => {
+      cy.switchToExtensionPermissionWindow().then(() => {
+        cy.disconnectWallet().then(taskCompleted => {
+          expect(taskCompleted).to.be.true;
+        });
+      });
```

Comment:
nit: is there anything more we can do to fully verify that the Dapp has disconnected?